docs/_docs/configuration/default.md

```diff
@@ -73,9 +73,6 @@ liquid:
 rdiscount:
   extensions        : []
 
```

Comment:
While we're at it, the RDiscount example config could probably also be removed.

```suggestion
```

Reply:
That's not needed in this PR. I'll merge the original `docs-40` branch after rebasing it so that the original contributions find their way to `master`